.azure-pipelines/MacOS-CI.yml

```diff
@@ -33,7 +54,7 @@ jobs:
       export ONNX_BUILD_TESTS=1
       export DEBUG=${onnx_debug}
       export ONNX_ML=${onnx_ml}
-      export CMAKE_ARGS="-DONNX_USE_LITE_PROTO=ON -DONNXIFI_DUMMY_BACKEND=ON"
+      export CMAKE_ARGS='-DONNXIFI_DUMMY_BACKEND=ON -DONNX_USE_LITE_PROTO='${onnx_lite}
```

Comment:
I dont think it is still working : 

ONNX version          : 1.7.0
--   ONNX NAMESPACE        : ONNX_NAMESPACE_FOO_BAR_FOR_CI
--   ONNX_BUILD_TESTS      : ON
--   ONNX_BUILD_BENCHMARKS : OFF
--   ONNX_USE_LITE_PROTO   : 
--   ONNXIFI_DUMMY_BACKEND : ON

Reply:
may be try something like this : 

if [ "${onnx_lite}" == "ON" ]; then
  export CMAKE_ARGS="${CMAKE_ARGS} -DONNX_USE_LITE_PROTO=ON"
fi